src/ubx.h

```diff
@@ -62,10 +62,12 @@
 #define UBX_CLASS_ACK		0x05
 #define UBX_CLASS_CFG		0x06
 #define UBX_CLASS_MON		0x0A
-#define UBX_CLASS_RTCM3	0xF5
+#define UBX_CLASS_EVENT     0x0D
+#define UBX_CLASS_RTCM3	0xF5 /**< This is undocumented (?) */
 
 /* Message IDs */
 #define UBX_ID_NAV_POSLLH	0x02
+#define UBX_ID_NAV_EVENT    0X03
```

Comment:
the format is not uniform,or just a display issue on github web?

Reply:
Just a display error on the web page